applications/FluidDynamicsApplication/custom_processes/weighted_divergence_calculation_process.h

```diff
@@ -0,0 +1,338 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
```

Comment:
Header format.

Reply:
What do you mean? In my Visual Studio it seems well formatted

Reply:
Done